spec/acceptance/nodesets/default.yml

```diff
@@ -60,5 +60,5 @@ HOSTS:
 
 CONFIG:
   log_level: verbose
-  type: foss
+  type: aio
```

Comment:
I should probably add this and ensuring the gemfile contains Trevor's beaker patch to the 1695 instructions